app/views/thredded/shared/_page.html.erb

```diff
@@ -1,6 +1,7 @@
 <div id="thredded--container" class="thredded--main-container <%= yield(:thredded_page_id) %><%= ' thredded--is-moderator' if moderatable_messageboards_ids.present? %>" data-thredded-page-id="<%= yield(:thredded_page_id) %>">
   <%= render 'thredded/shared/header' %>
   <%= render 'thredded/shared/flash_messages' %>
+  <div id="thredded--definitions" data-thredded-root="<%= thredded.root_path%>"></div>
```

Comment:
yes to `data-thredded-root-url` (and may as well make it a url, rather than a path)

and yes to defining it on `#thredded--container`.

Do you think maybe worth making thredded--container a helper as it's rather hard to read in erb? (I'll try this out as a commit)